### PR TITLE
Create Neogeo-CD.md

### PR DESCRIPTION
**Pull request by Unknown-programer**
added neogeo cd to the docs

### CODE REVIEW

**Comment by cmitu** [comment]
A few general comments on this new doc page:

 - it's missing FBneo as emulator.
 - extensions supported by NeoCD are just `.cue` and `.chd`, there's no `.zip` (see https://github.com/libretro/libretro-core-info/blob/master/neocd_libretro.info). 
 - Don't use `$biosdir`, please add the full path instead

File: docs/Neogeo-CD.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+***
+_ The Neo Geo CD is the second home video game console of SNK Corporation's Neo Geo family, released on September 9, 1994, four years after its cartridge-based equivalent. The system was originally priced at US$399, or £399. It was discontinued in 1997 with sales of 570,000 systems._
```
Comment (cmitu) [inline docs/Neogeo-CD.md:2]: I don't care about the price, but if you want to mention it was cheapter than the original Neogeo AES, that's ok.

File: docs/Neogeo-CD.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+***
+
+| Emulator | Rom Folder | Extension | BIOS |  Controller Config |
+| :---: | :---: | :---: | :---: | :---: |
+| [lr-neocd](https://github.com/libretro/neocd_libretro) | neogeo  | .chd .cue .zip | uni-bioscd.rom | /opt/retropie/configs/neogeo/retroarch.cfg |
```
Comment (cmitu) [inline docs/Neogeo-CD.md:8]: Add `fbneo`. Remove `.zip` as supported file.

File: docs/Neogeo-CD.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Emulator: [lr-neocd](https://github.com/libretro/neocd_libretro) 
+
+**lr-neocd**  is a complete rewrite of NeoCD from scratch in modern C++11. It is designed with accuracy and portability in mind rather than being all about speed like the the older versions.
+
+## ROMS
+Accepted File Extensions: **.cue** **.chd** **.zip**
+
+Place your Neogeo CD ROMs in
```
Comment (cmitu) [inline docs/Neogeo-CD.md:17]: Please your Neogeo _CDROM images_ ....

File: docs/Neogeo-CD.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+```
+/home/pi/RetroPie/roms/neogeo
+```
+## BIOS
+You will need a minimum of two BIOS files (eg. ng-lo.rom, uni-bioscd.rom) which should be copied to $biosdir/neocd.
```
Comment (cmitu) [inline docs/Neogeo-CD.md:22]: Why 2 BIOS files are needed ? The core documentation only mentions one. Also, as said in the general comments, expand `$biosdir`.

Comment (Unknown-programer) [reply docs/Neogeo-CD.md:22]: I was quoting the installation script for neocd https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/libretrocores/lr-neocd.sh

Comment (cmitu) [reply docs/Neogeo-CD.md:22]: Sorry, but I don't follow. The ROMs folder is specified completely with the path, so is the BIOS folder in the other emulator/ports pages. Please do so for this page also.

File: docs/Neogeo-CD.md (hunk 2 above)
Comment (Unknown-programer) [reply docs/Neogeo-CD.md:8]: I used a zip file on my own unmodified system and it worked fine.

**Comment by cmitu** [comment]
@Unknown-programer look, did you even played any Neogeo CD games ? You don't seem to be familiar with the system and just copy-pasting stuff from various places, but without any details that would help a RetroPie user. I.e. location of BIOS files is obviously wrong (should be under 'neocd' sub-folder), there's a missing BIOS file for `neocd` and there's no mention on how `fbneo` works (and not, `fbneo` doesn't work with `.zip` or `.7z` CD images).

**Comment by Unknown-programer** [comment]
> @Unknown-programer look, did you even played any Neogeo CD games ? You don't seem to be familiar with the system and just copy-pasting stuff from various places, but without any details that would help a RetroPie user. 

Yes I have played the games on my system, all of which I have tried proved functional. Addmitedly I am not extremely familiar with the system and am doing the best I can with what information I can find. As for copying and pasting, I wanted to keep the information accurate. I apologize for my mistakes that I have made and I will continue to correct them.

**Comment by cmitu** [comment]
> Yes I have played the games on my system, all of which I have tried proved functional. Addmitedly I am not extremely familiar with the system and am doing the best I can with what information I can find. 

 I'm starting to doubt that, given you've not provided the correct BIOS folder or the correct BIOS files needed for `neocd` (you don't seem to have used FBNEo though).

> As for copying and pasting, I wanted to keep the information accurate. 

Yes, but sometimes things need to be explained and expaned for newcomers. The info may be accurate, but it's lacking any context for a RetroPie installation (i.e. BIOS folder location, which BIOS files are needed, ROM format).

> I apologize for my mistakes that I have made and I will continue to correct them until they meet your approval.

Sure, but my time is not infinite and if your corrections are just taking my suggestions and adding them to the document - blindly - then I may as well close this and add the info myself.

How about this - if you add the correct info for FBNeo, then I'll accept the PR and do any modifications/correction afterwards. This means you have to play the game with FBNeo (have the correct file(s), BIOS in place, select the `-cd` emulator from `runcommand` menu).

**Comment by cmitu** [comment]
I guess not.

**Comment by Unknown-programer** [comment]
I apologize for my late response my schedule has been intense recently. I will submit a new pull request when I am able with fbneo information.On Feb 11, 2024, at 3:28 PM, Cristi Mitrana ***@***.***> wrote:﻿
Closed #148.

**Pull request closed without merging**